x-pack/legacy/plugins/snapshot_restore/public/app/components/policy_form/steps/step_settings.tsx

```diff
@@ -0,0 +1,450 @@
+                      if (!pattern.trim().length) {
+                        return;
+                      }
+                      const newPatterns = [...indexPatterns, pattern];
+                      setIndexPatterns(newPatterns);
+                      updatePolicyConfig({
+                        indices: newPatterns.join(','),
+                      });
+                    }}
+                    onChange={(patterns: Array<{ label: string }>) => {
+                      const newPatterns = patterns.map(({ label }) => label);
+                      setIndexPatterns(newPatterns);
+                      updatePolicyConfig({
+                        indices: newPatterns.join(','),
+                      });
+                    }}
+                  />
+                )}
+              </EuiFormRow>
+            </Fragment>
+          )}
+        </Fragment>
+      </EuiFormRow>
+    </EuiDescribedFormGroup>
+  );
+
+  const renderIgnoreUnavailableField = () => (
+    <EuiDescribedFormGroup
+      title={
+        <EuiTitle size="s">
+          <h3>
+            <FormattedMessage
+              id="xpack.snapshotRestore.policyForm.stepSettings.ignoreUnavailableDescriptionTitle"
+              defaultMessage="Ignore unavailable indices"
+            />
+          </h3>
+        </EuiTitle>
+      }
+      description={
+        <FormattedMessage
+          id="xpack.snapshotRestore.policyForm.stepSettings.ignoreUnavailableDescription"
+          defaultMessage="Ignores indices that are unavailable when taking the snapshot. Otherwise, the entire snapshot will fail."
+        />
+      }
+      idAria="policyIgnoreUnavailableDescription"
+      fullWidth
+    >
+      <EuiFormRow
+        hasEmptyLabelSpace
+        describedByIds={['policyIgnoreUnavailableDescription']}
+        fullWidth
+      >
+        <EuiSwitch
+          label={
+            <FormattedMessage
+              id="xpack.snapshotRestore.policyForm.stepSettings.ignoreUnavailableLabel"
+              defaultMessage="Ignore unavailable indices"
+            />
+          }
+          checked={config.ignoreUnavailable}
+          onChange={e => {
+            updatePolicyConfig({
+              ignoreUnavailable: e.target.checked,
+            });
+          }}
+        />
+      </EuiFormRow>
+    </EuiDescribedFormGroup>
+  );
+
+  const renderPartialField = () => (
+    <EuiDescribedFormGroup
+      title={
+        <EuiTitle size="s">
+          <h3>
+            <FormattedMessage
+              id="xpack.snapshotRestore.policyForm.stepSettings.partialDescriptionTitle"
+              defaultMessage="Allow partial indices"
+            />
+          </h3>
+        </EuiTitle>
+      }
+      description={
+        <FormattedMessage
+          id="xpack.snapshotRestore.policyForm.stepSettings.partialDescription"
+          defaultMessage="Allows snapshots of indices with primary shards that are unavailable. Otherwise, the entire snapshot will fail."
```

Comment:
@gchaps Same question here, should this be "Otherwise, the entire snapshot will fail if any primary shards are unavailable"?